src/main/java/co/cask/gcp/gcs/GCSConfigHelper.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.gcp.gcs;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * A helper class for GCS configs
+ */
+public final class GCSConfigHelper {
+
+  private static final String FILE_SYSTEM_PATH_PREFIX = "gs://";
+
+  /**
+   * Returns the given path as it is if it starts with {@link GCSConfigHelper#FILE_SYSTEM_PATH_PREFIX} else appends it
+   * in the beginning.
+   *
+   * @param path the path
+   * @return {@link URI} for the given path
+   * @throws IllegalArgumentException if the given path is not valid
+   */
+  public static URI getPath(String path) {
+    try {
+      URI uri = new URI(path);
+      if (uri.getScheme() != null && (!uri.getScheme().equalsIgnoreCase("gs") || uri.getAuthority() == null)) {
+        throw new URISyntaxException(path, "Scheme must be 'gs' and a bucket name must be specified.");
```

Comment:
this isn't actually a syntax exception. Just throw the IllegalArgumentException with the same message, as that's what this is -- an illegal user input.